.markdownlint.yaml

```diff
@@ -0,0 +1,11 @@
+# MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md013.md
+MD013: false
+
+# MD037/no-space-in-emphasis : Spaces inside emphasis markers : https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md037.md
```

Comment:
Some of these checks are not necessary as they would catch only a handful of cases. I'll amend the sources and remove them and repush.